### PR TITLE
ACT: Take into account async keyword when implementing missing fn

### PR DESCRIPTION
**Pull request by mchernyavsky**
Fixes https://github.com/intellij-rust/intellij-rust/issues/4510.

### CODE REVIEW

File: src/test/kotlin/org/rust/ide/refactoring/implementMembers/ImplementMembersHandlerTest.kt [hunk 1]
```diff
@@ -29,14 +29,14 @@ class ImplementMembersHandlerTest : RsTestBase() {
 
     private fun invokeVia(actionInvoker: () -> Unit) {
         checkByText("""
-            trait T { fn f1(); }
+            trait T { async unsafe fn f1(); }
```
Comment (Undin) [inline src/test/kotlin/org/rust/ide/refactoring/implementMembers/ImplementMembersHandlerTest.kt]: Maybe it's better to create separate test for this case?

Comment (mchernyavsky) [reply src/test/kotlin/org/rust/ide/refactoring/implementMembers/ImplementMembersHandlerTest.kt]: Fixed.

File: src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt [hunk 2]
```diff
@@ -36,7 +36,7 @@ class RsFileStub : PsiFileStubImpl<RsFile> {
 
     object Type : IStubFileElementType<RsFileStub>(RsLanguage) {
         // Bump this number if Stub structure changes
-        override fun getStubVersion(): Int = 180
+        override fun getStubVersion(): Int = 181
```
Comment (Undin) [inline src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: Update it please according to the latest value in master

Comment (mchernyavsky) [reply src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: Fixed.

**Pull request merged**